it/thrift/build.gradle

```diff
@@ -0,0 +1,42 @@
+dependencies {
+    // Thrift
+    api 'org.apache.thrift:libthrift'
+    api 'javax.annotation:javax.annotation-api'
+
+    // Jetty, for testing TServlet interoperability.
+    testImplementation 'org.eclipse.jetty:jetty-webapp'
+    testImplementation 'org.eclipse.jetty.http2:http2-server'
+
+    // Dropwizard and Prometheus, for testing metrics integration
+    testImplementation 'io.dropwizard.metrics:metrics-core'
+    testImplementation 'io.micrometer:micrometer-registry-prometheus'
+    testImplementation 'io.prometheus:simpleclient_common'
+}
+
+ext {
+    enableThriftFullCamel()
+}
+
+// Use the sources from ':thrift0.13'.
+// NB: We should never add these directories using the 'sourceSets' directive because that will make
+//     them added to more than one project and having a source directory with more than one output directory
+//     will confuse IDEs such as IntelliJ IDEA.
+tasks.compileJava.source "${rootProject.projectDir}/thrift0.13/src/main/java"
+tasks.processResources.from "${rootProject.projectDir}/thrift0.13/src/main/resources"
+tasks.sourcesJar.from "${rootProject.projectDir}/thrift0.13/src/main/java"
+tasks.sourcesJar.from "${rootProject.projectDir}/thrift0.13/src/main/resources"
+tasks.javadoc.source "${rootProject.projectDir}/thrift0.13/src/main/java"
```

Comment:
Don't we need to copy the source code from `thrift0.13`? How about using `implementation project(':thrift0.13')` instead?

Reply:
It's OK. I do this because the comment let me a little confuse about this.